src/main/java/domain/LottoMarket.java

```diff
@@ -3,14 +3,16 @@
 public class LottoMarket {
 
     private final int price;
+    private final int PassiveLottoCount;
```

Comment:
```suggestion
    private final int passiveLottoCount;
```

여기도 price가 꼭 필요하지 않을 수도 있을 것 같습니다!